docs/content/en/docs/core-components/api-reporter.md

```diff
@@ -0,0 +1,168 @@
+|execution_id       |UUID                   |string             |The id of the execution
+|playbook_id        |UUID                   |string             |The id of the CACAO playbook executed by the execution
+|started            |timestamp              |string             |The time at which the execution of the playbook started
+|ended              |timestamp              |string             |The time at which the execution of the playbook ended (if so)
+|status             |execution-status-enum  |string             |The current [status](#execution-stataus) of the execution
+|status_text        |explanation            |string             |A natural language explanation of the current status or related info
+|error              |error                  |string     |Error raised along the execution of the playbook at execution level
+|step_results       |step_results           |dictionary         |Map of step-id to related [step execution data](#step-execution-data)
+|request_interval   |seconds                |integer            |Suggests the polling interval for the next request (default suggested is 5 seconds).
+
+
+##### Step execution data
+|field              |content                |type               | description |
+| ----------------- | --------------------- | ----------------- | ----------- |
+|execution_id       |UUID                   |string             |The id of the execution of the playbook where the step resides
+|step_id            |UUID                   |string             |The id of the step being executed
+|started            |timestamp              |string             |The time at which the execution of the step started
+|ended              |timestamp              |string             |The time at which the execution of the step ended (if so)
+|status             |execution-status-enum  |string             |The current [status](#execution-stataus) of the execution of this step
+|status_text        |explanation            |string             |A natural language explanation of the current status or related info
+|error              |error                  |string             |Error raised along the execution of the step
+|variables          |cacao variables        |dictionary         |Map of [cacao variables](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256555) handled in the step (both in and out) with current values and definitions
+
+##### Execution stataus 
+Table from [Cyentific RNI workflow Status](https://github.com/cyentific-rni/workflow-status/blob/main/README.md#21-refined-execution-status-enumeration)
+**Vocabulary Name:** `execution-status-enum`
+| Property Name | Description|
+| :--- | :--- |
+| successfully_executed | The workflow step was executed successfully (completed). |
+|failed| The workflow step failed. |
+|ongoing| The workflow step is in progress. |
+|server_side_error| A server-side error occurred. |
```

Comment:
I've sugested await_user_input https://github.com/cyentific-rni/workflow-status/pull/3 if that goes true we need to add that here